optuna_dashboard/_named_objectives.py

```diff
@@ -22,6 +27,17 @@ def set_objective_names(study: optuna.Study, names: list[str]) -> None:
           study = optuna.create_study(directions=["minimize", "minimize"])
           set_objective_names(study, ["val_loss", "flops"])
     """
+
+    if hasattr(study, "set_metric_names"):
+        # TODO(c-bata): List this warning on errors.rst in docs.
```

Comment:
I will address this todo item after #427 is merged since `errors.rst` will be added in it.

Reply:
Can we address this in this PR because #427 will be merged soon?

Reply:
Yes, that makes sense 👍 I will address this in this PR.